crypto/modes/asm/aes-gcm-armv8_64.pl

```diff
@@ -12,7 +12,46 @@
 # derived from https://github.com/ARM-software/AArch64cryptolib, original
 # author Samuel Lee <Samuel.Lee@arm.com>. The module is, however, dual
 # licensed under OpenSSL and CRYPTOGAMS licenses depending on where you
-# obtain it. For further details see http://www.openssl.org/~appro/cryptogams/.
+# obtain it. For further details see https://github.com/dot-asm/cryptogams.
+# 
+# or
```

Comment:
I don't understand the purpose of `or` here. Does it just mean "The cryptogams licence is"?

(The `or` comes after "Written by ...". I could understand

line 4: "Licensed under..."
line 17: "or"
Lines 19ff: The CRYPTOGAMS licence

But that's not how this reads - instead it's more "Written by ..." or (CRYPTOGAMS licence))

Reply:
Yeah, instead of `or` I would use: `The CRYPTOGAMS license:`

However it is also unclear to me if this license should be copied here verbatim. The license is present on the https://github.com/dot-asm/cryptogams link so IMO not necessary here.

Reply:
Agreed - this is an alternative to the OpenSSL licence - the details aren't necessarily appropriate for the OpenSSL source code to hold